authorization/role/service/role_management_service.go

```diff
@@ -87,7 +87,7 @@ func (s *roleManagementServiceImpl) Assign(ctx context.Context, assignedBy uuid.
 	// privileges for the resource, otherwise the invitation workflow should be used instead
 	assignments := make(map[uuid.UUID][]uuid.UUID)
 
-	var existingRoleIDs []uuid.UUID
+	existingRoles := []rolerepo.IdentityRole{}
```

Comment:
just curious, what's the benefit of an array of `IdentityRole` vs `UUID` ?

Reply:
No benefit, simply that `IdentityRoleRepository.FindIdentityRolesByIdentityAndResource()` returns an array of `IdentityRole`.

Reply:
ok